doc/source/terrain.md

```diff
@@ -42,169 +178,212 @@ The aspect describes the orientation of strongest slope.
+
+$$
+\theta = arctan(\frac{p}{q}),
+$$
 
-```{image} basic_examples/images/sphx_glr_plot_terrain_attributes_002.png
-:width: 600
+with $p$ and $q$ defined in the slope section.
+
+```{warning}
+A north aspect represents the upper direction of the Y axis in the coordinate reference system of the
+input, {attr}`xdem.DEM.crs`, which might not represent the true north.
 ```
 
-```{eval-rst}
-.. minigallery:: xdem.terrain.aspect
-        :add-heading:
+```{code-cell} ipython3
+aspect = dem.aspect()
+aspect.plot(cmap="twilight", cbar_title="Aspect (°)")
 ```
 
+(hillshade)=
 ## Hillshade
 
 {func}`xdem.DEM.hillshade`
 
 The hillshade is a slope map, shaded by the aspect of the slope.
-The slope map is a good tool to visualize terrain, but it does not distinguish between a mountain and a valley.
-It may therefore be slightly difficult to interpret in mountainous terrain.
-Hillshades are therefore often preferable for visualizing DEMs.
 With a westerly azimuth (a simulated sun coming from the west), all eastern slopes are slightly darker.
-This mode of shading the slopes often generates a map that is much more easily interpreted than the slope map.
+This mode of shading the slopes often generates a map that is much more easily interpreted than the slope.
+
+The hillshade $hs$ is directly based on the slope $\alpha$ and aspect $\theta$, and thus also varies between the method of [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) (default) and that of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107), and
+is often scaled between 1 and 255:
 
-As the hillshade is directly based on the slope and aspect, it varies between the method of [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) (default) and that of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107).
+$$
+hs = 1 + 254 \left[ sin(alt) cos(\alpha) + cos(alt) sin(\alpha) sin(2\pi - azim - \theta) \right],
+$$
+
+where $alt$ is the shading altitude (90° = from above) and $azim$ is the shading azimuth (0° = north).
 
 Note, however, that the hillshade is not a shadow map; no occlusion is taken into account so it does not represent "true" shading.
 It therefore has little analytic purpose, but it still constitutes a great visualization tool.
 
-```{image} basic_examples/images/sphx_glr_plot_terrain_attributes_003.png
-:width: 600
-```
-
-```{eval-rst}
-.. minigallery:: xdem.terrain.hillshade
-        :add-heading:
+```{code-cell} ipython3
+hillshade = dem.hillshade()
+hillshade.plot(cmap="Greys_r", cbar_title="Hillshade")
 ```
 
+(curv)=
 ## Curvature
 
 {func}`xdem.DEM.curvature`
 
-The curvature map is the second derivative of elevation, which highlights the convexity or concavity of the terrain.
+The curvature is the second derivative of elevation, which highlights the convexity or concavity of the terrain.
 If a surface is convex (like a mountain peak), it will have positive curvature.
```

Comment:
I thought we had discussed it before, but it's the opposite. convex = valley, concave = summit.

Reply:
Uuuh what do you mean here @adehecq? A convex surface is summit-like while a concave surface is valley-like.

Reply:
Thanks @erikmannerfelt, I also find several sources backing the current statements!
https://www.britannica.com/science/valley/Valley-cross-profiles
https://avalanche.ca/glossary/terms/convex-slope
Leaving as is :wink: